### PR TITLE
Activate venv at server start and remove the check from functions

### PR DESCRIPTION
**Pull request by SilasMarvin**

### CODE REVIEW

File: pgml-extension/src/lib.rs [hunk 1]
```diff
@@ -24,6 +24,7 @@ extension_sql_file!("../sql/schema.sql", name = "schema");
 #[cfg(not(feature = "use_as_lib"))]
 #[pg_guard]
 pub extern "C" fn _PG_init() {
+    bindings::python::activate().expect("Error setting python venv");
```
Comment (levkk) [inline pgml-extension/src/lib.rs:27]: This works? I ask because each connection forks from the postmaster. It gets the venv set correctly? :O

Comment (SilasMarvin) [reply pgml-extension/src/lib.rs:27]: It works for me locally. I can test from multiple different connections.

Comment (SilasMarvin) [reply pgml-extension/src/lib.rs:27]: Can confirm tested from three different connections and it worked

Comment (levkk) [reply pgml-extension/src/lib.rs:27]: Well, that was easy.

**Review by levkk — APPROVED** [review]

**Review by montanalow — APPROVED** [review]
🎉

**Pull request merged**